fda-model/structures/Inventory.cs

```diff
@@ -181,9 +181,12 @@ private void LoadStructuresFromSourceFiles()
 
                 int fid = GetRowValueForColumn<int>(row, _map.StructureIDCol, defaultMissingValue);               
                 double val_struct = GetRowValueForColumn<double>(row,_map.StructureValueCol, defaultMissingValue);
-
-                string st_damcat = GetRowValueForColumn(row, _map.DamageCatagory, "NA");
                 string occtype = GetRowValueForColumn(row, _map.OccTypeCol, "NA");
+                string st_damcat = GetRowValueForColumn(row, _map.DamageCatagory, "NA");
```

Comment:
I know i had this line in the image i pasted, but since we don't think that "GetRowValueForColumn" will ever work for the dam cat, lets just assign it "NA" on this line and then let it go into the "if".